### PR TITLE
[EDX-142] Add an overview to integrations

### PR DESCRIPTION
**Pull request by m-hulbert**
> _NOTE TO REVIEWERS_ - please **do not review PRs in the `DRAFT` state**, as the PR may change substantially before it is ready to review. Thanks.

## Description

This PR adds an overview page to the integrations section.

See [JIRA](https://ably.atlassian.net/browse/EDX-142) for further information. 

## Review

View the new integrations overview to review this PR.

### CODE REVIEW

**Review by Sapples — APPROVED** [review]
One comment that is down to personal preference/taste, otherwise LGTM.

File: content/general/integrations.textile [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+meta_description: "Integrations enable external services to send data to Ably channels, and for Ably events to send their data to external services."
+meta_keywords: "integrations, integration, integrate, stream, external service, webhook, webhooks, functions"
+section: general
+index: 0
+jump_to:
+  - Events via webhooks#webhooks
+  - Continuous streaming#streaming
+---
+
+Integrations enable data to be sent from Ably to external services and for external services to send data to Ably channels.
```
Comment (Srushtika) [inline content/general/integrations.textile:12]: Maybe add the word 'channels' for both directions?

File: content/general/integrations.textile [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+Ably events can be used to trigger webhook requests to external services such as "IFTTT":https://ifttt.com/ and "Zapier":https://zapier.com/ or invoke functions such as "Amazon Lambda functions":https://aws.amazon.com/lambda/. Ably events can also be streamed to external streaming and queuing services such as "Amazon Kinesis":https://aws.amazon.com/kinesis/ and "Amazon SQS":https://aws.amazon.com/sqs/.
```
Comment (Srushtika) [inline content/general/integrations.textile:14]: Think we can skip the 'functions' in Amazon Lambda

Comment (Srushtika) [reply content/general/integrations.textile:14]: It's confusing to see 'events' when the rest of the documentation talks in terms of 'messages'. Could you add some context so readers can understand what we mean by 'events' here?

File: content/general/integrations.textile [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+Data can be sent from an external service into Ably channels. Configure an "incoming webhook":#incoming-webhooks in Ably to use in an external service, or use the "Ably Kafka Connector":/ to stream data from Kafka into Ably to distribute messages to client devices. 
+
+h2(#webhooks). Events via webhooks
```
Comment (Srushtika) [inline content/general/integrations.textile:18]: It'll be useful to provide some context around events vs continuous streaming before getting into each section

File: content/general/integrations.textile [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+jump_to:
+  - Events via webhooks#webhooks
+  - Continuous streaming#streaming
+---
+
+Integrations enable data to be sent from Ably to external services and for external services to send data to Ably channels.
+
+Ably events can be used to trigger webhook requests to external services such as "IFTTT":https://ifttt.com/ and "Zapier":https://zapier.com/ or invoke functions such as "Amazon Lambda functions":https://aws.amazon.com/lambda/. Ably events can also be streamed to external streaming and queuing services such as "Amazon Kinesis":https://aws.amazon.com/kinesis/ and "Amazon SQS":https://aws.amazon.com/sqs/.
+
+Data can be sent from an external service into Ably channels. Configure an "incoming webhook":#incoming-webhooks in Ably to use in an external service, or use the "Ably Kafka Connector":/ to stream data from Kafka into Ably to distribute messages to client devices.
```
Comment (Srushtika) [inline content/general/integrations.textile:16]: It feels like we have inbound and outbound integrations mixed up leading to confusion

**Review by Srushtika — CHANGES_REQUESTED** [review]
Thanks, @m-hulbert - I've added a few comments but would be good to jump on a 10min call to discuss further. It would be good to align the content in this page with the IA plan

**Review by Srushtika — APPROVED** [review]
Looks good to me, thanks @m-hulbert . @tbedford are you able to have a look at this and merge it in while Mark is away?

**Pull request merged**